packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -110,38 +110,53 @@ Accounts.prototype.create = function create(entropy) {
 };
 
 Accounts.prototype.privateKeyToAccount = function privateKeyToAccount(privateKey, targetAddressRaw) {
-  var { privateKey: prvKey, address, isHumanReadable } = utils.parsePrivateKey(privateKey)
+  let {nonDecoupledAccount: account, klaytnWalletKeyAddress} = this.getNonDecoupledAccount(privateKey)
```

Comment:
getNonDecoupledAccount is little bit weird to me.
What about using 'LegacyAccount' or something else?

Reply:
Ah , okay ! I think getLegacyAccount sounds better !!  :) I modify the name.